controllers/space/space_controller.go

```diff
@@ -308,6 +363,22 @@ func (r *Reconciler) setStatusTerminatingFailed(logger logr.Logger, space *toolc
 	return cause
 }
 
+func (r *Reconciler) setStatusChangeFailed(logger logr.Logger, space *toolchainv1alpha1.Space, cause error) error {
+	if err := r.updateStatus(
+		space,
+		toolchainv1alpha1.Condition{
+			Type:    toolchainv1alpha1.ConditionReady,
+			Status:  corev1.ConditionFalse,
+			Reason:  toolchainv1alpha1.SpaceStatusUpdateFailedReason,
+			Message: cause.Error(),
+		}); err != nil {
+		logger.Error(cause, "unable to compute hash for NSTemplateTier with name '%s'", space.Spec.TierName)
```

Comment:
I guess this is not the right error to log?

Reply:
this block of code was removed